app/src/main/java/com/android/wardrobeManager/ClosetActivity.java

```diff
@@ -1,31 +1,78 @@
 package com.android.wardrobeManager;
 
 import android.content.Intent;
+import android.opengl.Visibility;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 import android.view.View;
 import android.widget.Button;
+import android.widget.TextView;
 
 public class ClosetActivity extends AppCompatActivity {
 
+
+    boolean shirtsScrollVisible = false;
+    boolean shoesScrollVisible = false;
+    boolean shortsScrollVisible = false;
+
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_closet);
 
-        configureBackButton();
+    }
+
+    public void backToMainScreen(View view) {
+        startActivity(new Intent(ClosetActivity.this, StartActivity.class));
+    }
+
+    public void toggleShirtVisibility(View view) {
+
+        if (shirtsScrollVisible) {
+            shirtsScrollVisible = false;
+            findViewById(R.id.shirtScrollView).setVisibility(View.GONE);
+            ((TextView)findViewById(R.id.shirtsToggleView)).setText(">");
```

Comment:
Do you have to use 'getViewById'? Isn't the view you need passed into this method as view? I have never done android before so I am probably wrong on this, but double check.

Reply:
After further testing you do need to use getViewById because I am passing in the view of either the button or the view that the button is in. I'm not sure which one because I am not using the view variable anyway, but the button does not work unless the method has the View parameter. The findViewById is searching for the horizontal scroll box, which is not passed into the view variable, so that's why I need to use it.

Reply:
Okay, cool! Good job tracking that down!